src/app/frontend/chrome/chrome.scss

```diff
@@ -84,3 +84,9 @@ a {
 [role='tabpanel'] {
   transition: none;
 }
+
+.material-icons {
+  &.kd-error {
+    color: $secondary;
```

Comment:
I dont think this should be `$secondary`. It is a coincidence that our secondary color is red. It was pink not so long ago. 

This should have color of {{warn-A700}} as defined, e.g., here: https://github.com/angular/material/blob/1f997951b5d480d04ea7d356107837e279d6a271/src/components/input/input-theme.scss#L70